moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -178,12 +182,20 @@ const useColumns = ({
         field: "funding_status_id",
         editable: true,
         width: 200,
-        renderCell: ({ value }) =>
+        valueFormatter: (value) =>
           getLookupValueByID(
             data["moped_fund_status"],
             "funding_status",
             value
           ),
+        sortComparator: (v1, v2) =>
+          getLookupValueByID(
+            data["moped_fund_status"],
+            "funding_status",
+            v1
+          ).localeCompare(
+            getLookupValueByID(data["moped_fund_status"], "funding_status", v2)
```

Comment:
the funding status wasnt actually sorting correctly, bc previously it was sorting by this fields `value` which is the `id` of the status, so it was numerically sorting the ids and not alphabetically sorting the lookup values users see on the front end